sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f

```diff
@@ -3960,15 +3941,26 @@ SUBROUTINE INITPOST_GFS_NEMS_MPIIO(iostatusAER)
       do K = 1, nbin_oc
        if ( K == 1) VarName='oc1wtc'
        if ( K == 2) VarName='oc2wtc'
-       VcoordName='atmos col'
+       VcoordName='sfc'
        l=1
        call assignnemsiovar(im,jsta,jend,jsta_2l,jend_2u               &
                            ,l,nrec,fldsize,spval,tmp                   &
                            ,recname,reclevtyp,reclev,VarName,VcoordName&
                            ,ocsv(1,jsta_2l,K))
       enddo
+! retrieve MIE AOD
+       VarName='maod'
+       VcoordName='sfc'
+       l=1
+       call assignnemsiovar(im,jsta,jend,jsta_2l,jend_2u               &
+                           ,l,nrec,fldsize,spval,tmp                   &
```

Comment:
delete one space to align with the previous lines

Reply:
did it